script/build

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+BUILD_DIRECTORY="build"
+CONFIGURATION=Release
+
+if [[ -z $TRAVIS_XCODE_WORKSPACE ]]; then
+	echo "\$TRAVIS_XCODE_WORKSPACE is not set. Using \`ReactiveCocoa.xcworkspace\`"
+	TRAVIS_XCODE_WORKSPACE="ReactiveCocoa.xcworkspace"
+fi
+
+if [[ -z $TRAVIS_XCODE_SCHEME ]]; then
+	echo "\$TRAVIS_XCODE_SCHEME is not set. Using \`ReactiveCocoa-Mac\`"
+	TRAVIS_XCODE_SCHEME="ReactiveCocoa-Mac"
+fi
+
+if [[ -z $XCODE_ACTION ]]; then
+      echo "\$XCODE_ACTION is not set. Using \`build\`"
+      XCODE_ACTION="build"
+fi
+
+if [[ -z $XCODE_SDK ]]; then
+      echo "\$XCODE_SDK is not set. Using \`macosx\`"
+      XCODE_SDK="macosx"
+fi
```

Comment:
I'm not wild about duplicating these is in the script. Can we just require them to be set?